.github/workflows/osrm-backend.yml

```diff
@@ -627,10 +529,38 @@ jobs:
         ./src/benchmarks/packedvector-bench
         ./src/benchmarks/rtree-bench ../test/data/monaco.osrm.ramIndex ../test/data/monaco.osrm.fileIndex ../test/data/monaco.osrm.nbg_nodes
         popd
-    - name: Run Node package tests only
+
+    - name: Use Node 16
+      if: ${{ matrix.NODE_PACKAGE_TESTS_ONLY == 'ON' && matrix.ENABLE_APPLE_SILICON != 'ON' }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+    - name: Run Node package tests on Node 16
       if: ${{ matrix.NODE_PACKAGE_TESTS_ONLY == 'ON' && matrix.ENABLE_APPLE_SILICON != 'ON' }}
       run: |
+        node --version
         npm run nodejs-tests
+    - name: Use Node 18
```

Comment:
Paranoia check that resulting binary is compatible with any Node version we currently support. I am going to revisit it and probably try to do it in more elegant and less verbose way(probably we can use `nvm` instead of GitHub actions in order to switch Node versions)